deploy/helm-chart/README.md

```diff
@@ -130,30 +130,30 @@ kubectl delete -f https://github.com/kubernetes-sigs/gateway-api/releases/downlo
 
 ## Configuration
 
-The following tables lists the configurable parameters of the NGINX Kubernetes Gateway chart and their default values.
+The following tables lists the configurable parameters of the NGINX Gateway Fabric chart and their default values.
 
 | Parameter | Description | Default Value |
 |-----------|-------------|---------------|
-| `nginxGateway.image.repository` | The repository for the NGINX Kubernetes Gateway image. | ghcr.io/nginxinc/nginx-kubernetes-gateway |
-| `nginxGateway.image.tag` | The tag for the NGINX Kubernetes Gateway image. | edge |
-| `nginxGateway.image.pullPolicy` | The `imagePullPolicy` for the NGINX Kubernetes Gateway image. | Always |
-| `nginxGateway.gatewayClassName` | The name of the GatewayClass for the NGINX Kubernetes Gateway deployment. | nginx |
+| `nginxGateway.image.repository` | The repository for the NGINX Gateway Fabric image. | ghcr.io/nginxinc/nginx-gateway-fabric |
```

Comment:
Do you think it's ok to leave these as `nginxGateway`?

Reply:
I'm okay with it.